core/GlobalState.h

```diff
@@ -251,6 +251,9 @@ class GlobalState final {
     // The error code is appended to this string.
     std::string errorUrlBase;
 
+    // If 'true', do not print error sections
+    bool noErrorSections = false;
```

Comment:
Can you make this be

```suggestion
    bool includeErrorSections = true;
```

I generally don't like double negations: `!noErrorSections`.

Reply:
Sure! Do you mean changing it only in the global state or everywhere, including the CLI argument?

Reply:
I'm fine either way. You can either leave it as is, or change it to `--include-error-sections` with a default value of `true` using cxxopts.

Reply:
Switched the variable name for the global state, but kept the CLI option as `-no-error-sections`.